### PR TITLE
Russian Translation

### PR DESCRIPTION
**Pull request by LIMUNADE**
Made a full Russian translation.

### CODE REVIEW

**Comment by LIMUNADE** [comment]
Eh, I don't know what's failed in there tbh.

**Comment by LIMUNADE** [comment]
@lukeocodes there is something wrong with checks. I fixed them and on the next check bot telling me to fix what I changed.

File: lang/ru.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+      error: 'Ошибка',
+      retry: 'Проверить еще раз?',
+      success:
+        'Этот проект пока ещё принимает участие в <strong>Hacktoberfest</strong>... на данный момент.',
+      failure:
+        'Этот проект ещё не учавствует в <strong>Hacktoberfest</strong>... на данный момент.',
+      results_are_in: 'Подводятся результаты для',
+      cta: 'Уверены?',
+      noRepoFound:
+      'Предыдущие проверки репозитория не найдены ... пока что!',
```
Comment (lukeocodes) [inline lang/ru.js:66]:
```suggestion
      noRepoFound: 'Предыдущие проверки репозитория не найдены ... пока что!',
```

File: ru.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+      title: 'Принимает ли проект участие в <strong>Hacktoberfest?</strong>',
+      error: 'Ошибка',
+      retry: 'Снова проверить?',
+      success:
+        'Этот проект пока ещё принимает участие в <strong>Hacktoberfest</strong> ... на данный момент.',
+      failure:
+        'Этот проект ещё не учавствует в <strong>Hacktoberfest</strong> ... на данный момент.',
+      results_are_in: 'Подводятся результаты для',
+      cta: 'Уверены?',
+      noRepoFound:
```
Comment (lukeocodes) [inline ru.js:65]:
```suggestion
      noRepoFound:
```

File: .github/lang/README.ru.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    <td align="center"><a href="https://leobia.github.io/"><img src="https://avatars2.githubusercontent.com/u/26444579?v=4" width="100px;" alt=""/><br /><sub><b>Leonardo</b></sub></a><br /><a href="https://github.com/lukeocodes/hacktoberfest-checker/issues?q=author%3Aleobia" title="Bug reports">🐛</a> <a href="https://github.com/lukeocodes/hacktoberfest-checker/commits?author=leobia" title="Code">💻</a></td>
+  </tr>
+</table>
+
+<!-- markdownlint-enable -->
+<!-- prettier-ignore-end -->
+<!-- ALL-CONTRIBUTORS-LIST:END -->
+
+Этот проект следует [все-участники](https://github.com/all-contributors/all-contributors) спецификации. Любая помощь приветствуется!
+
```
Comment (lukeocodes) [inline .github/lang/README.ru.md:101]:
```suggestion

```

**Comment by lukeocodes** [comment]
@LIMUNADE I've added some suggestions I think it will pass now :) Thanks so much for this!

**Comment by LIMUNADE** [comment]
Glad to help

**Comment by lukeocodes** [comment]
Can you accept those suggestions and we'll try and get it passing

**Comment by LIMUNADE** [comment]
> 
> 
> Can you accept those suggestions and we'll try and get it passing

Yes, certainly. I'm sort of new to github

**Comment by lukeocodes** [comment]
@LIMUNADE I did one last change, it was a stray space haha. excellent work if you're new to GitHub, really excellent. thanks so much :)

**Comment by lukeocodes** [comment]
@all-contributors add @LIMUNADE for translations

**Pull request merged**